ts/Series/HLC/HLCSeries.ts

```diff
@@ -0,0 +1,497 @@
+/* *
+ *
+ *  (c) 2010-2021 Pawel Lysy
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type HLCSeriesOptions from './HLCSeriesOptions';
+import type { StatesOptionsKey } from '../../Core/Series/StatesOptions';
+import type SVGAttributes from '../../Core/Renderer/SVG/SVGAttributes';
+import type SVGElement from '../../Core/Renderer/SVG/SVGElement';
+import type SVGPath from '../../Core/Renderer/SVG/SVGPath';
+import HLCPoint from './HLCPoint.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+const {
+    seriesTypes: {
+        column: ColumnSeries
+    }
+} = SeriesRegistry;
+import U from '../../Core/Utilities.js';
+const {
+    extend,
+    merge
+} = U;
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * The hlc series type.
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.hlc
+ *
+ * @augments Highcharts.Series
+ */
+class HLCSeries extends ColumnSeries {
+
+    /* *
+     *
+     *  Static Properties
+     *
+     * */
+
+    /**
+     * An HLC chart is a style of financial chart used to describe price
+     * movements over time. It displays high, low and close values per
+     * data point.
+     *
+     * @sample stock/demo/hlc/
+     *         HLC chart
+     *
+     * @extends      plotOptions.column
+     * @excluding    borderColor, borderRadius, borderWidth, crisp, stacking,
+     *               stack
+     * @product      highstock
+     * @optionparent plotOptions.hlc
+     */
+    public static defaultOptions: HLCSeriesOptions = merge(ColumnSeries.defaultOptions, {
```

Comment:
If we inherited OHLCSeries instead, only the `tooltip.pointFormat` would be different.